README.md

```diff
@@ -1,10 +1,31 @@
+# Newcomers checklist
+
+Soft intro:
+
+- [ ] Introduction to the team
+- [ ] Code for Poznań introduction
+- [ ] Projects overview
+
+Tech intro:
+
+- [ ] Environment setup (Ubuntu/MacOS)
+- [ ] GitHub account
+- [ ] Invitation to the GitHub organization
+- [ ] Gitter access
+- [ ] GitHub flow
+- [ ] Zenhub
+- [ ] SSH key
+- [ ] Docker installation
+- [ ] Docker-compose installation
+- [ ] Task assignment
+
 # Structure of files
 
-Currently we have two folders - first for organization stuff, second for technical tutorials. 
+Currently we have two folders - first for organization stuff, second for technical tutorials.
 
 If you are new here, please start with reading `getting_to_know_us.md` within `organization_onboarding` folder.
 
-If you need any assistance with technical stuff like Github flow or Docker installation, go to `technical_onboarding` folder. 
+If you need any assistance with technical stuff like Github flow or Docker installation, go to `technical_onboarding` folder.
 
 # Naming of the files
```

Comment:
Jakich plików?

Reply:
Look at the whole file - click expand on left. 

There were no changes made here (aside from typo), I was only removing whitespace at the end of the lines.